.github/workflows/test.js.yml

```diff
@@ -36,6 +36,13 @@ jobs:
           PGDATABASE: frap-dev
           PGUSER: frap
           PGPASSWORD: frap
+      - run: yarn migrate-fra
```

Comment:
why do we have this in the tests? I dont think it should be here

Reply:
For now, we use it to test against some data
Should we remove and remove tests that use data?